charts/langwatch/templates/langwatch_nlp/secrets.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.autogen.secretNames.langwatch_nlp | default (printf "%s-langwatch-nlp-secrets" .Release.Name) }}
+  labels:
+    {{- include "langwatch.labels" . | nindent 4 }}
+type: Opaque
+data:
+  {{- $existingSecret := lookup "v1" "Secret" .Release.Namespace (.Values.autogen.secretNames.langwatch_nlp | default (printf "%s-langwatch-nlp-secrets" .Release.Name)) }}
+  {{- if $existingSecret }}
+  # Reuse existing secret data
+  {{- range $key, $value := $existingSecret.data }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)